test/functional/outputs/syslog/forward_to_syslog_test.go

```diff
@@ -131,30 +131,6 @@ var _ = Describe("[Functional][Outputs][Syslog] Functional tests", func() {
 			Expect(getProcID(fields)).To(Equal("myproc"))
 			Expect(getMsgID(fields)).To(Equal("mymsg"))
 		})
-		It("should take values of appname, procid, messageid from record", func() {
```

Comment:
Let's try to enable these tests by introducing the JSON parse filter to demonstrate how this functionality should be work

Reply:
5.9 don't have JSON filter, i will do it in 6.x